From 92de6069e75d9981c2f9ab0f7cbd2970236d51e3 Mon Sep 17 00:00:00 2001
From: Vadim Pasternak <vadimp@nvidia.com>
Date: Mon, 17 Jul 2023 15:40:08 +0000
Subject: [PATCH 19/78] Extend driver to support Infineon Digital Multi-phase
 XDPE1A2G7 device.

From telemetry perspective device is almost the same as XDPE15284, but
does not support READ_EIN (86h) and READ_EOUT (87h) commands.

Signed-off-by: Vadim Pasternak <vadimp@nvidia.com>
---
 drivers/hwmon/pmbus/xdpe152c4.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/hwmon/pmbus/xdpe152c4.c b/drivers/hwmon/pmbus/xdpe152c4.c
index 7f3b31d..089f268 100644
--- a/drivers/hwmon/pmbus/xdpe152c4.c
+++ b/drivers/hwmon/pmbus/xdpe152c4.c
@@ -44,6 +44,7 @@ static int xdpe152_probe(struct i2c_client *client)
 }
 
 static const struct i2c_device_id xdpe152_id[] = {
+	{"xdpe1a2g7"},
 	{"xdpe152c4"},
 	{"xdpe15284"},
 	{}
@@ -52,6 +53,7 @@ static const struct i2c_device_id xdpe152_id[] = {
 MODULE_DEVICE_TABLE(i2c, xdpe152_id);
 
 static const struct of_device_id __maybe_unused xdpe152_of_match[] = {
+	{.compatible = "infineon,xdpe1a2g7"},
 	{.compatible = "infineon,xdpe152c4"},
 	{.compatible = "infineon,xdpe15284"},
 	{}
-- 
2.8.4

